addon/lib/auth-request.js

```diff
@@ -19,36 +20,26 @@ function pingServer(config, title, data, addon) {
       }
     });
 
-    getMe(headers, config.BASE_URL, function(response) {
-      let id = '';
-      if (response.json) {
-        id = response.json.id;
-      }
-
-      request({
-        url: config.BASE_URL + '/api/metrics/?format=json',
-        content: JSON.stringify(formatEvent(config.TESTPILOT_PREFIX, title, id, data, addon)),
-        headers: headers,
-        onComplete: function(resp) { // eslint-disable-line no-unused-vars
-          // console.error(resp);
-        }
-      }).post();
-    });
+    cb(headers);
   });
 }
 
-function getMe(headers, BASE_URL, cb) {
```

Comment:
We aren't using it anymore in this pr. How do you feel about adding it back in once we need it for feature flipping? That way we don't have unused code in here